src/autodiscover.ts

```diff
@@ -1,14 +1,14 @@
 import { TemplateSettings } from './types'
-import autodiscoverTemplate from './views/autodiscover.xml'
+import autoDiscoverTemplate from './views/autodiscover.xml'
 import swig from 'swig-templates'
 
+// Microsoft Outlook
 export default (settings: TemplateSettings) =>
   async (req: Request): Promise<Response> => {
-    // Microsoft Outlook / Apple Mail
-
     const schema: string | null =
       req.bodyXml?.Autodiscover?.Request?.AcceptableResponseSchema
-    const xmlns: string | null = req.bodyXml?.Autodiscover?.$?.xmlns
+    const xmlns: string | null = schema !== null ? schema :
```

Comment:
FWIW, the test is still passing in CI, so...that's weird.  Did you com

This may not be the correct logic, actually.  It's causing the test to fail because the "schema" is the _response_ schema, while the namespace is the namespace for the `request`.  The original behavior was to use the NS _if_ the schema wasn't found.  This needs a better test lattice: test for missing AcceptableResponseSchema (fallback to NS) vs use the AcceptableResponseSchema as provided.

Looking at what is being sent back in the request,

Reply:
Is it passing the test with this branch? Because I updated a few other things around this. 

If you change this in your branch, does it still pass?

Reply:
I just pulled (your latest as of 11pm PDT) and the tests pass, so I'm going to go ahead and add the additional tests to cover those cases before moving on to swapping-out the templating engine for `mustache`.